docs/contents/components/forms/year-picker/index.ja.mdx

```diff
@@ -28,59 +28,95 @@ import { YearPicker } from "@yamada-ui/calendar"
 
 ```tsx
 <VStack>
-  <YearPicker variant="outline" placeholder="outline" />
-  <YearPicker variant="filled" placeholder="filled" />
-  <YearPicker variant="flushed" placeholder="flushed" />
-  <YearPicker variant="unstyled" placeholder="unstyled" />
+  <For each={["outline", "filled", "flushed", "unstyled"]}>
+    {(variant, index) => (
+      <YearPicker key={index} variant={variant} placeholder={variant} />
+    )}
+  </For>
 </VStack>
 ```
 
 カレンダーのバリアントを変更する場合は、`variant`を設定します。
 
 ```tsx
 <VStack>
-  <YearPicker placeholder="solid" calendarVariant="solid" />
-  <YearPicker placeholder="subtle" calendarVariant="subtle" />
+  <For each={["solid", "subtle"]}>
+    {(variant, index) => (
+      <YearPicker key={index} variant={variant} placeholder={variant} />
+    )}
+  </For>
 </VStack>
 ```
 
 ### サイズを変更する
 
 ```tsx
 <VStack>
-  <YearPicker placeholder="extra small size" size="xs" />
-  <YearPicker placeholder="small size" size="sm" />
-  <YearPicker placeholder="medium size" size="md" />
-  <YearPicker placeholder="large size" size="lg" />
+  <For each={["solid", "subtle"]}>
+    {(variant, index) => (
+      <YearPicker key={index} variant={variant} placeholder={variant} />
+    )}
+  </For>
 </VStack>
 ```
 
 カレンダーのサイズを変更する場合は、`calendarSize`を設定します。
 
 ```tsx
 <VStack>
-  <YearPicker placeholder="small size" calendarSize="sm" />
-  <YearPicker placeholder="medium size" calendarSize="md" />
-  <YearPicker placeholder="large size" calendarSize="lg" />
+  <For
+    each={[
```

Comment:
@kaehehehe 
It seems that using `<For />` has actually reduced readability. Why don't we try replacing `<For />` with something else?